### PR TITLE
Batch trigger

### PR DESCRIPTION
**Pull request by jpatel531**
WDYT @zimbatm

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -10,4 +10,6 @@ function trigger(pusher, channels, eventName, data, socketId, callback) {
   pusher.post({ path: '/events', body: event }, callback);
 }
 
-exports.trigger = trigger;
+exports.triggerBatch = function(pusher, batch, callback) {
+	pusher.post({ path: '/batch_events', body: { batch: batch } }, callback);
```
Comment (zimbatm) [inline lib/events.js]: if you want to allow passing arbitrary "data" attributes, they need to be encoded to JSON before posting

Comment (zimbatm) [inline lib/events.js]: probably in the other triggerBatch method

**Comment by jpatel531** [comment]
Publishing closes #35

File: lib/events.js [hunk 2]
```diff
@@ -10,4 +10,13 @@ function trigger(pusher, channels, eventName, data, socketId, callback) {
   pusher.post({ path: '/events', body: event }, callback);
 }
 
-exports.trigger = trigger;
+exports.triggerBatch = function(pusher, batch, callback) {
+  for (var i = 0; i < batch.length; i++) {
+    batch[i].data = ensureJSON(batch[i].data);
+  }
+	pusher.post({ path: '/batch_events', body: { batch: batch } }, callback);
```
Comment (zimbatm) [inline lib/events.js]: tab !

**Pull request merged**